controller/eks-cluster-config-handler.go

```diff
@@ -878,7 +876,7 @@ func (h *Handler) updateUpstreamClusterState(upstreamSpec *eksv1.EKSClusterConfi
 			&eks.UpdateClusterConfigInput{
 				Name: aws.String(config.Spec.DisplayName),
 				ResourcesVpcConfig: &eks.VpcConfigRequest{
-					EndpointPublicAccess: config.Spec.PublicAccess,
+					EndpointPublicAccess:  config.Spec.PublicAccess,
```

Comment:
nit: extra whitespace?

Reply:
bump

Reply:
This was a `go fmt` thing so that value of the parameters line up visually. Is it important that I revert this?

Reply:
na